liquidcore/site/settings.py

```diff
@@ -162,11 +162,14 @@ def get_interval(spec):
     'REFRESH_TOKEN_GRACE_PERIOD_SECONDS': 120,  # recommended from docs
     "OIDC_ENABLED": True,
     "OIDC_ISS_ENDPOINT": LIQUID_URL,
+    "OIDC_RSA_PRIVATE_KEY": os.environ.get('OIDC_RSA_PRIVATE_KEY'),
```

Comment:
not needed or set?

Reply:
not needed anymore, leftover of test, it can depend on algo chosen though, so far we use HS256